apps/src/code-studio/components/progress/UnitCalendarButton.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import i18n from '@cdo/locale';
+import Button from '@cdo/apps/templates/Button';
+import UnitCalendarDialog from './UnitCalendarDialog';
+
+export default class UnitCalendarButton extends React.Component {
```

Comment:
I think it would be good to add this component as part of the work to update the script overview page.

Reply:
When you do that I don't think you will need the storybook component for this anymore